packages/myst-transforms/src/math.ts

```diff
@@ -43,16 +43,28 @@ function labelMathNodes(file: VFile, node: Math | InlineMath) {
   const label = match[1];
   const normalized = normalizeLabel(label);
   if (node.type === 'math' && normalized) {
-    if (node.enumerated === false) {
-      fileWarn(file, `Labelling an unnumbered math node with "\\label{${label}}"`, {
-        node,
-        source: TRANSFORM_NAME,
-        ruleId: RuleId.mathLabelLifted,
-      });
+    if (node.label) {
+      fileWarn(
+        file,
+        `Math node is already labeled "${node.label}" - ignoring inline "\\label{${label}}"`,
```

Comment:
Given the edge case of a math node with both (1) `\label` in the content and (2) `mystTarget` label: This extra warning ensures behavior is unchanged now that `mystTargetsTransform` happens before these math transforms.